### PR TITLE
chore: prepare 1.7.0

### PR DESCRIPTION
**Pull request by sbouchet**
## What is the purpose of this change? What does it change?
prepare 1.7.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,15 +27,30 @@ The extension also supports OpenShift running on Azure, AWS, and any others supp
 
 ## New features
 
+### Install Helm Charts on the current cluster within the IDE
+
+We're Happy to announce that we've implemented a new action that provide the capability to install
+a [Helm](https://helm.sh/) chart into the current cluster.
+
+Below a small demo using Heml stable repo (https://charts.helm.sh/stable) and installing kuberos chart.
```
Comment (mohitsuman) [inline README.md:35]:
```suggestion
Below is a small demo of the workflow: installing `kuberos` chart from the Helm stable repo (https://charts.helm.sh/stable) 
```

File: README.md [hunk 2]
```diff
@@ -27,15 +27,30 @@ The extension also supports OpenShift running on Azure, AWS, and any others supp
...
+
+![](images/1.7.0/helm.gif)
+
+There is no need to install anything prior to use Helm. By default if there is no repository configured, the openshift
```
Comment (mohitsuman) [inline README.md:39]:
```suggestion
There is no need to install anything prior to use Helm. By default if there is no repository configured, the OpenShift
```

File: README.md [hunk 3]
```diff
@@ -27,15 +27,30 @@ The extension also supports OpenShift running on Azure, AWS, and any others supp
...
+helm repository will be use (https://charts.openshift.io/).
```
Comment (mohitsuman) [inline README.md:40]:
```suggestion
Helm repository will be used (https://charts.openshift.io/).
```

File: src/main/resources/META-INF/plugin.xml [hunk 4]
```diff
@@ -17,6 +17,13 @@
     ]]></description>
 
   <change-notes><![CDATA[
+  <p><b>1.7.0</b></p>
+  <ul>
+  <li>Switch to odo 3.15.0</li>
+  <li>Added an action to install Helms Charts in cluster</li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:23]:
```suggestion
  <li>Added an action to install Helm Charts in the connected cluster</li>
```

**Review by mohitsuman — CHANGES_REQUESTED** [review]
Typos and wording fixes

**Review by adietish — APPROVED** [review]

**Pull request merged**